test/phi.test.js

```diff
@@ -0,0 +1,39 @@
+const { phi } = require("../lib");
+
+describe("phi", () => {
+  test("phi(1) equals 1", () => {
```

Comment:
should this be "phi(7) equals 1"?

Reply:
No it should be phi(1), thanks for pointing this out. I'll fix this